### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by AlecKC** [illinois-cs241/illinois-cs241.github.io#42]
As I am taking a course that relates assistive technology, I've come to realize just how important it is that our course page be accessible to those with accessibility needs (the DOJ cares).  Thus, we need to go through our course page and make sure it is very accessible.

[WCAG 2.0](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines): A guideline (standard) for accessible electronic information.

[Letter to Universities from DOJ](http://www2.ed.gov/about/offices/list/ocr/letters/colleague-20100629.html)

[Accessibility Policy](http://www.ssbbartgroup.com/blog/accessibility-policy-series-3-phase-1-policy-development/): We should have one on our course website

**Comment by bchong95** [comment]
I skimmed those links and I think we met those requirements.

Here is some tool I used to make sure that we don't have any glaring errors: http://wave.webaim.org/report#/http://cs241.cs.illinois.edu/

Please let me know if there are any specifics things that you would like changed to make the site more accessible

**Comment by bchong95** [comment]
You can even try copy and pasting an entire page from the site into a screen reader and it seems to work out well:

http://ttsreader.com/

I am going to close the issue. Please reopen if there is something that you superficially want to see changed.
